### PR TITLE
reference architecture deployment

### PR DESCRIPTION
**Pull request by msokoloff1**
* Simplify pipeline to always run remote
* Update config to support remote deployment
* Read secrets from google cloud instead of env vars
* Add basic deployment scripts

Still requires manual config to set secrets and the rest of the env. A subsequent pr will be created to make deployment / env config as easy as running a single script.

### CODE REVIEW

File: examples/integrations/gcp/src/coordinator/pipelines/images/bounding_box.py [hunk 1]
```diff
@@ -5,48 +5,42 @@
...
 
-from pipelines.types import Pipeline, CustomJob, JobStatus, JobState, Job
+from pipelines.types import Pipeline, JobStatus, JobState, Job
 
 logger = logging.getLogger("uvicorn")
 
 
-class BoundingBoxETL(CustomJob):
+class BoundingBoxETL(Job):
+    container_name = "gcr.io/sandbox-5500/training-repo/bounding_box_etl"
```
Comment (Juanbuhler) [inline examples/integrations/gcp/src/coordinator/pipelines/images/bounding_box.py:14]: what's the plan longer term for these container URIs? will they be passed as arguments? right now a user would have to edit these both in the docker-compose file and in all of the etl classes, right?

**Review by Juanbuhler — COMMENTED** [review]
Could you add at least a mention of how to use deploy.sh in the README?

File: examples/integrations/gcp/src/coordinator/pipelines/images/bounding_box.py (hunk 1 above)
Comment (msokoloff1) [reply examples/integrations/gcp/src/coordinator/pipelines/images/bounding_box.py]: oops. yeah the project name should come from an env var. I just updated it. Thanks!

**Comment by msokoloff1** [comment]
> Could you add at least a mention of how to use deploy.sh in the README?

I made some updates to the readme. Instead of explaining all of the steps that need to be taken to prep for deployment via clicking the UI, I am going to follow up in a new PR to #1 automate the deployment and #2 explain how to use the script.

**Comment by Juanbuhler** [comment]
Looking good!

**Pull request merged**